test/src/tests.js

```diff
@@ -1253,10 +1264,13 @@ describe('Adwords forwarder', function () {
                     },
                 ];
 
-                window.dataLayer.length.should.eql(1);
+                window.dataLayer.length.should.eql(2);
                 window.dataLayer[0][0].should.equal('consent');
                 window.dataLayer[0][1].should.equal('default');
-                window.dataLayer[0][2].should.deepEqual(expectedDataLayer[2]);
+                window.dataLayer[0][2].should.deepEqual(expectedDataLayer1[2]);
```

Comment:
I had similar feedback for alex here - you can probably refactor these 2 checks for the [0] and [1] index to be something like:

window.dataLayer[0].should.deepEqual(expectedDataLayer1)
window.dataLayer[1].should.deepEqual(expectedDataLayer2)